In [1]:
from openai import OpenAI

client = OpenAI(api_key='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')

import datasets

dataset=datasets.load_dataset("paws", "labeled_final")

/home/ansh/.local/lib/python3.10/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html
  from .autonotebook import tqdm as notebook_tqdm


In [2]:
sentences1=[dataset['train'][i]['sentence1'] for i in range(len(dataset['train']))]
sentences2=[dataset['train'][i]['sentence2'] for i in range(len(dataset['train']))]
labels=[str(dataset['train'][i]['label']) for i in range(len(dataset['train']))]
print((labels[0]))

0


In [3]:
messages = [
    {"role": "system", "content": "You are a kind helpful assistant."},
]
messages.append(
    {"role": "user", "content": "Reply only in 0(NO) or 1(YES) for the following questions. DONOT USE ANY OTHER WORDS. "}
)

chat_completion = client.chat.completions.create(
    messages=messages,
    model="gpt-3.5-turbo",
)
cont = chat_completion.choices[0].message.content
print(cont)
messages.append( {"role": "assistant", "content": cont})

Understood. I will reply only using 0 or 1. Please proceed with your questions.


In [4]:
import time

correct=0
times=100

outputList=[]

for i in range(times):
    
    messages.append({"role": "user", "content": "Do these two sentences mean the same thing?\n\n"+sentences1[i]+"\n"+sentences2[i]+"\n\n"})
    chat_completion = client.chat.completions.create(
        messages=messages,
        model="gpt-3.5-turbo",
    )
    cont = chat_completion.choices[0].message.content
    print(sentences1[i])
    print(sentences2[i])
    print(cont, labels[i])
    print()

    if cont==labels[i]:
        correct+=1
    print(correct/(i+1))

    outputList.append([sentences1[i], sentences2[i], cont, labels[i], correct/(i+1)])
    messages.pop()

In Paris , in October 1560 , he secretly met the English ambassador , Nicolas Throckmorton , asking him for a passport to return to England through Scotland .
In October 1560 , he secretly met with the English ambassador , Nicolas Throckmorton , in Paris , and asked him for a passport to return to Scotland through England .
0 0

1.0
The NBA season of 1975 -- 76 was the 30th season of the National Basketball Association .
The 1975 -- 76 season of the National Basketball Association was the 30th season of the NBA .
1 1

1.0
There are also specific discussions , public profile debates and project discussions .
There are also public discussions , profile specific discussions , and project discussions .
0 0

1.0
When comparable rates of flow can be maintained , the results are high .
The results are high when comparable flow rates can be maintained .
1 1

1.0
It is the seat of Zerendi District in Akmola Region .
It is the seat of the district of Zerendi in Akmola region .
1 1

1.0
William H

RateLimitError: Error code: 429 - {'error': {'message': 'Rate limit reached for gpt-3.5-turbo in organization org-HJQAJeQ4lvA87BJgCEdhyJC2 on requests per min (RPM): Limit 3, Used 3, Requested 1. Please try again in 20s. Visit https://platform.openai.com/account/rate-limits to learn more. You can increase your rate limit by adding a payment method to your account at https://platform.openai.com/account/billing.', 'type': 'requests', 'param': None, 'code': 'rate_limit_exceeded'}}

## First try with 5 queries

In Paris , in October 1560 , he secretly met the English ambassador , Nicolas Throckmorton , asking him for a passport to return to England through Scotland .

In October 1560 , he secretly met with the English ambassador , Nicolas Throckmorton , in Paris , and asked him for a passport to return to Scotland through England .

0 0


The NBA season of 1975 -- 76 was the 30th season of the National Basketball Association .

The 1975 -- 76 season of the National Basketball Association was the 30th season of the NBA .

1 1


There are also specific discussions , public profile debates and project discussions .

There are also public discussions , profile specific discussions , and project discussions .

0 0


When comparable rates of flow can be maintained , the results are high .

The results are high when comparable flow rates can be maintained .

1 1


It is the seat of Zerendi District in Akmola Region .

It is the seat of the district of Zerendi in Akmola region .

1 1

## Tried for 100 queries.
Rate Limit Exceeded in the middle

In Paris , in October 1560 , he secretly met the English ambassador , Nicolas Throckmorton , asking him for a passport to return to England through Scotland .
In October 1560 , he secretly met with the English ambassador , Nicolas Throckmorton , in Paris , and asked him for a passport to return to Scotland through England .
0 0

1.0
The NBA season of 1975 -- 76 was the 30th season of the National Basketball Association .
The 1975 -- 76 season of the National Basketball Association was the 30th season of the NBA .
1 1

1.0
There are also specific discussions , public profile debates and project discussions .
There are also public discussions , profile specific discussions , and project discussions .
0 0

1.0
When comparable rates of flow can be maintained , the results are high .
The results are high when comparable flow rates can be maintained .
1 1

1.0
It is the seat of Zerendi District in Akmola Region .
It is the seat of the district of Zerendi in Akmola region .
1 1

1.0
William Henry Henry Harman was born on 17 February 1828 in Waynesboro , Virginia , where his parents were Lewis and Sally ( Garber ) Harman .
William Henry Harman was born in Waynesboro , Virginia on February 17 , 1828 . His parents were Lewis and Sally ( Garber ) Harman .
1 1

1.0
Bullion Express - concept is being introduced new store in Dallas , Texas in Preston Center opened .
2011-DGSE Bullion Express concept is introduced , new store opened in Preston Center in Dallas , Texas
0 0

1.0
With a discrete amount of probabilities Formula 1 with the condition formula 2 and Formula 3 any real number , the Tsallis is defined as entropy as
Given a discrete set of probabilities formula _ 1 with the condition formula _ 2 , and formula _ 3 any real number , the Tsallis entropy is defined as
1 1

1.0
The Soviet Union maintained an embassy in Oslo and a consulate in Barentsburg , while Norway maintained a message in Moscow .
The Soviet Union maintained an embassy in Moscow and a consulate in Barentsburg , while Norway maintained a message in Oslo .
0 0

1.0
Vocabulary even went to Brazil through leaving Portuguese settlers with some Macanese and Chinese settlers .
Vocabulary even went to Brazil by leaving Macanese and Chinese settlers with some Portuguese settlers .
1 0

0.9
Kabir Suman recorded several albums under the name of Suman Chattopaddhyay or Suman Chatterjee between 1992 and 1999 .
Suman Chatterjee , recorded a number of albums between 1992 and 1999 under the name Suman Chattopaddhyay or Kabir Suman .
0 0

0.9090909090909091
He was a scholar in Metaphysical Literature , Theology and Classical sciences .
He was a scholar in metaphysical literature , theology , and classical science .
1 1

0.9166666666666666
The city sits at the confluence of the Snake River with the great Weiser River , which marks the border with Oregon .
The city lies at the confluence of the Snake River and the Great Weiser River , which marks the border with Oregon .
1 1

0.9230769230769231
He has been trained by his grandfather , Nick Dakin , and is now trained by Geoff Barraclough .
He has been trained by his grandfather , Geoff Barraclough , and is now coached by Nick Dakin .
0 0

0.9285714285714286
The Austrian school assumes that the subjective choices of individuals , including individual knowledge , time , expectations , and other subjective factors , cause all economic phenomena .
The Austrian school assumes that the subjective choices of individuals , including subjective knowledge , time , expectation , and other individual factors , cause all economic phenomena .
1 0

0.8666666666666667
Werder 's forces invested Belfort and reached the city on 3 November .
Werder 's troops invested Belfort and reached the city on November 3 .
1 1

0.875
The kBox facilitates both isometric and concentric contractions as well as eccentric training .
The kBox facilitates eccentric as well as concentric contractions and isometric training .
0 0

0.8823529411764706
The first five weapons were delivered in the first half of 1916 , with a total of 57 barrels and 56 cars completed by the end of the war .
The first five weapons were delivered in the first half of 1916 . A total of 57 barrels and 56 carriages were completed by the end of the war .
0 1

0.8333333333333334
Elizabeth II was an ancestor of Queens Edzard II and Beatrix of the Netherlands .
Edzard II was an ancestor of the Queens Elizabeth II and the Beatrix of the Netherlands .
0 0

0.8421052631578947
The friendship between him and Duncan ended at a club meeting in 1951 when the two disagreed at an annual meeting and Duncan reported that Greaves said :
The friendship between him and Duncan ended in 1951 at a club meeting , when the two did not agree at an annual meeting , and Duncan reported that Greaves said :
1 1

0.85
Pluto was classified as the planet when the Grand Tour was proposed and was launched at the time `` New Horizons '' .
Note : Pluto was classified as a planet when the Grand Tour was launched and at the time `` New Horizons '' was proposed .
1 0

0.8095238095238095
For their performances in the game , quarterback Jameis Winston and defensive back P. J. Williams were named the game 's most valuable players .
Quarterback P. J. Williams and Defensive Back Jameis Winston were named the most valuable players of the game for their performances in the game .
0 0

0.8181818181818182
Shaffer Creek is a tributary of the Raystown Branch Juniata River ( Brush Creek ) in Bedford County , Pennsylvania , United States .
Shaffer Creek is an tributary of Brush Creek ( Raystown Branch Juniata River ) in Bedford County , Pennsylvania in the United States .
0 1

0.782608695652174
Kevin Spacey ( Henry Drummond ) and David Troughton ( Matthew Harrison Brady ) starred in a 2009 revival at The Old Vic in London .
Kevin Spacey ( Henry Drummond ) and David Troughton ( Matthew Harrison Brady ) played in a resume in 2009 at the Old Vic London .
0 0

0.7916666666666666
Briggs later met Briggs at the 1967 Monterey Pop Festival , where Ravi Shankar was also performing , with Eric Burdon and The Animals .
Briggs met Briggs later at the Monterey Pop Festival of 1967 , where Ravi Shankar also performed with Eric Burdon and The Animals .
0 1

0.76
Laura Myntti was born in Salt Lake City and lived in Sioux City , Iowa and San Diego before settling in Minnesota in 1968 .
Born in Minnesota , Laura Myntti lived in Sioux City , Iowa and San Diego , before settling in Salt Lake City in 1968 .
0 0

0.7692307692307693
The female lead role was played by Cortez in `` Ali Baba and the Sacred Crown '' , directed by Erminio Salvi .
Cortez played the female lead in `` Ali Baba and the Sacred Crown '' , directed by Erminio Salvi .
0 1

0.7407407407407407
She worked and lived in Stuttgart , Berlin ( Germany ) and in Vienna ( Austria ) .
She worked and lived in Germany ( Stuttgart , Berlin ) and in Vienna ( Austria ) .
0 1

0.7142857142857143
Akshuat dendropark ( Russian : Акшуатский дендропарк ) is a natural monument ( Ulyanovsk Oblast protected areas )
Akshuat dendropark ( Russian : Акшуатский дендропарк ) is a natural monument ( Protected areas of Ulyanovsk Oblast )
1 1

0.7241379310344828
The Little Jocko River flows across the Saint Lawrence River and the Ottawa River to the Jocko River .
The Little Jocko River flows via the Saint Lawrence River and the Ottawa River to the Jocko River .
1 1

0.7333333333333333
In 1951 , he died and retired in 1956 .
He died in 1951 and retired in 1956 .
0 1

0.7096774193548387
WORHP , also referred to as eNLP ( European NLP Solver ) , is a mathematical software library for numerically solving continuous , nonlinear optimization problems on a large scale .
WORHP , also referred to as eNLP ( European NLP solver ) by ESA , is a mathematical software library for solving continuous large scale nonlinear optimization problems numerically .
0 1

0.6875
On January 17 , 2000 , there were reports of the RSS and some BJP - hardliners threatening to restart the party .
On 17 January 2000 , there were reports of the BJP and some RSS hard-liners threatening to restart the party .
1 0

0.6666666666666666
The leaves are generally 1.5-4 mm long and 0.2-0.7 mm wide .
The leaves are usually 1.5-4 mm long and 0.2-0.7 mm wide .
1 1

0.6764705882352942
BA transferred the former BCal routes to Heathrow to Tokyo and Saudi Arabia .
BA relocated the former BCal routes to Tokyo and Saudi Arabia to Heathrow .
1 0

0.6571428571428571
In addition to Michael Boddicker , and Patrick Moraz , the album includes musical contributions from Diana Hubbard , John Goodsall , Chick Corea , Stanley Clarke .
In addition to Diana Hubbard , the album contains musical contributions from Chick Corea , Stanley Clarke , John Goodsall , Michael Boddicker and Patrick Moraz .
0 0

0.6666666666666666
These geographically correspond to the traditional Glasgow District , South , Edinburgh District , and North and Midlands districts .
These geographically roughly correspond to the traditional districts of Glasgow District , South , Edinburgh District and North and Midlands .
0 1

0.6486486486486487
Quintus Caecilius Metellus Macedonicus was the second son of Roman politician and general Lucius Caecilius Metellus Diadematus .
Quintus Caecilius Metellus Macedonicus was the second son of the Roman politician , General Lucius Caecilius Metellus Diadematus .
1 1

0.6578947368421053
He is trained by Andre Rozier and shares a gym with the former World Champion Daniel Jacobs .
He is trained by Daniel Jacobs and shares a gym with former world champion Andre Rozier .
0 0

0.6666666666666666
Here we view pseudo-differential operators as a generalization of differential operators .
We consider pseudo-differential operators here as a generalization of differential operators .
1 1

0.675
Although there are overwhelming social implications , there also seem to be regional financial patterns that can perpetuate this trend .
Although there are regional financial implications , there also seem to be overwhelming social patterns that perpetuate this trend .
0 0

0.6829268292682927
The museum building retains its European style of oriental modern architecture through the standing bricks on the southeast corner of the building and the classical image of the foyer .
The museum building retains its classic style of oriental modern architecture through the standing bricks on the southeast corner of the building and the European image of the foyer .
0 0

0.6904761904761905
Based on the city of Baltimore , only mentioned , never visited in the show .
Based on the city of Baltimore , only mentioned , has never visited in the show .
1 1

0.6976744186046512
Debbie Downer is a name of a fictional `` Saturday Night Live '' character that was debuted in 2004 and portrayed by Rachel Dratch .
Debbie Downer is a name of a fictional `` Saturday Night Live '' character who debuted in 2004 , and who was portrayed by Rachel Dratch .
1 1

0.7045454545454546
Taieri is a former parliamentary electorate in the Otago region of New Zealand , from 1866 to 1911 .
Taieri is a former parliamentary electorate in the New Zealand region of Otago , from 1866 to 1911 .
0 1

0.6888888888888889
This work caused him to trigger important reflections on the practices of molecular genetics and genomics at a time when this was not considered ethical .
This work led him to trigger ethical reflections on the practices of molecular genetics and genomics at a time when this was not considered important .
0 0

0.6956521739130435
Brockton is approximately 25 miles northeast of Providence , Rhode Island , and 30 miles south of Boston .
Brockton is located approximately 25 miles northeast of Providence , Rhode Island and 30 miles south of Boston .
0 1

0.6808510638297872
Seb Janiak is a French photographer and video director of Polish origin .
Seb Janiak is the French photographer and video director of Polish origin .
0 1

0.6666666666666666
Singer Angélique Kidjo and actor Djimon Hounsou were born in Cotonou , Benin .
Singer Djimon Hounsou and actor Angélique Kidjo were born in Cotonou in Benin .
0 0

0.673469387755102
In the 2015 regional elections , after Tosi was excluded by the Federal Party , Gidoni was elected to the Veneto Regional Council in the province of Belluno .
In the 2015 federal election , after Tosi was sidelined by the regional party , Gidoni was elected to the Regional Council of Veneto in the province of Belluno .
0 0

0.68
Ashley was born on 1 November 1986 and is a contemporary dancer from Los Angeles who originally grew up in Arizona .
Born on November 1 , 1986 , Ashley is a contemporary dancer from Los Angeles who was originally raised in Arizona .
1 1

0.6862745098039216
These include replicas at Ramat Shlomo in Israel and in Kfar Chabad in Jerusalem .
These include replicas in Ramat Shlomo in Jerusalem and in Kfar Chabad , Israel .
0 0

0.6923076923076923
Dorothy Kate Richmond , Frances Hodgkins , and Gwen Knight was a contemporary of Stewart .
Dorothy Kate Richmond , Frances Hodgkins and Gwen Knight was a Stewart contemporary .
0 1

0.6792452830188679
Hastings Ndlovu was buried with Hector Pieterson at Avalon Cemetery in Johannesburg .
Hastings Ndlovu , together with Hector Pieterson , was buried at the Avalon cemetery in Johannesburg .
1 1

0.6851851851851852
Davidson is a statue created by Will Rogers , which unveiled two versions in 1938 .
Will Rogers is a statue created by Jo Davidson , two versions of which were unveiled in 1938 .
0 0

0.6909090909090909
Other car manufacturers that have produced models with suicide doors include Citroën , Lancia , Opel , Panhard , Rover , Saab , Saturn , Skoda and Volkswagen .
Other car manufacturers which have produced models with suicide doors include Opel , Lancia , Citroën , Panhard , Rover , Saab , Saturn , Škoda , and Volkswagen .
1 1

0.6964285714285714
The first main bridge was completed in 1857 and the positioned bridge opened on 2 May 1859 by Prince Albert .
The first main span was positioned in 1857 and the completed bridge was opened by Prince Albert on 2 May 1859 .
0 0

0.7017543859649122
English also has many words , such as `` zillion '' , used informally to mean indefinite and fictitious but unspecified amounts ; see large numbers .
English also has many words , such as '' zillion `` , which are used informally to mean large but unspecified amounts , see indefinite and fictitious figures .
1 0

0.6896551724137931
His family moved from Brooklyn to Manhattan later on 110th Street and Amsterdam Avenue .
Later , his family moved from Brooklyn to 110th Street and Amsterdam Avenue in Manhattan .
0 0

0.6949152542372882
In 1991 he left Italy to work in the palliative care of cancer patients in Germany as well as in various countries of the Third World .
He left Germany in 1991 to work in the palliative care of cancer patients in Italy as well as in various Third World countries .
0 0

0.7
When Mustafa marched to Murad in Anatolia , Junayd was persuaded to confront him .
When Mustafa marched to confront Murad in Anatolia , Junayd was persuaded to desert him .
0 0

0.7049180327868853
The progress teacher Jürgen Reichen ( Swiss Education ) founded this method `` Writing to read '' 1982 .
The Swiss teacher Jürgen Reichen ( progressive education ) founded this `` writing to read '' method 1982 .
0 0

0.7096774193548387
After their departure from the Spear family in 1954 , Speer stopped singing and moved to Nashville in 1968 and back to Ohio after her husband had died .
After her departure from the Speer Family in 1954 , Speer stopped singing and moved to Ohio and back to Nashville in 1968 after her husband died .
0 0

0.7142857142857143
She has taken part in the second season of the most popular non - fiction controversial bengali reality show `` Bigg Boss Bangla '' .
She has participated in the second season of the most controversial non fiction popular bengali reality show `` Bigg Boss Bangla '' .
1 0

0.703125
It is covered with a natural vegetation of grassland of less fertile red clay bases , and Saltbush bushland on more fertile earths .
It is covered with a natural vegetation of grassland of less fertile red clay soils , and saltbush shrubland on more fertile earths .
1 1

0.7076923076923077
Jonté Buhl ( born April 4 , 1982 ) is a former Canadian professional football player who played for four years at the Edmonton Eskimos of the Canadian Football League .
Jonté Buhl ( born April 4 , 1982 ) is a Canadian former professional football cornerback who played four seasons for the Edmonton Eskimos of the Canadian Football League .
0 1

0.696969696969697
The continuous power density is determined by the product 's continuous torque density and the constant torque speed range of the electric machine .
The constant power density is determined by the product of the continuous torque density and the continuous torque speed range of the electric machine .
0 0

0.7014925373134329
She plays for Naisten Liiga of Åland United .
She plays for Naisten Liiga of the Åland United .
1 1

0.7058823529411765
Smaller small cross circuit pylons may have two single arms on one side and one on the other .
Smaller small cross pylons may have two single arms on one side and one on the other .
0 1

0.6956521739130435
`` The Timber '' was released in North America on February 27 , 2015 , Well Go USA Entertainment released it in Germany on October 6 , 2015 .
On February 27 , 2015 , `` The Timber '' was released in North America , Well Go USA Entertainment was released on October 6 , 2015 in Germany .
NO 1

0.6857142857142857
Although five issues of the series were printed , the project was finished without any of them being aborted .
Although five issues of the series were printed , the project was finished without any of them being cancelled .
0 1

0.676056338028169
As a songwriter , Lars Ankerstjerne has written songs for Nik & Jay , Burhan G and Rasmus Seebach .
Lars Ankerstjerne has written as songwriter for Nik 'Jay , Burhan G and Rasmus Seebach songs .
0 1

0.6666666666666666
This is also a shearing effect : when the focal length is larger , the shear effect is smaller .
This is also a shearing effect : when the focal length is smaller , the shearing effect is greater .
0 0

0.6712328767123288
In 1968 , the boardwalk Bowl followed the Tangerine Bowl , and the Pecan Bowl moved within Texas , from Abilene to Arlington .
In 1968 , the boardwalk Bowl followed Tangerine Bowl , and the Pecan Bowl moved from Abilene to Texas within Arlington .
0 0

0.6756756756756757
Concord is of imperfective type in the ergative aspect but perfective in the nominative aspect .
Concord is in the imperfective aspect of the nominative type , but supplementary in the perfective aspect .
0 0

0.68
The Soviet cosmonaut was Yuri Gagarin 's first air force pilot , also the first man in space .
The first cosmonaut was the Soviet air force pilot Yuri Gagarin , also the first person in space .
0 0

0.6842105263157895
On May 6 , 2016 it was announced that Palafox signed to National Premier Soccer League B of the New York Cosmos .
On 6 May 2016 , it was announced that Palafox had signed the National Premier Soccer League after New York Cosmos B .
0 0

0.6883116883116883
On 7 July 2011 , Russell was traded for Kris Russell on the Columbus Blue Jackets and joined his brother Michael Blunden with the Blue Jackets organization .
On July 7 , 2011 , Russell was traded to the Columbus Blue Jackets for Kris Russell . He joined his brother Michael Blunden with the Blue Jackets organization .
1 1

0.6923076923076923
A boy who survived the attack said that Oliveira selectively shot to kill girls while shooting boys only to immobilize them .
A boy who survived the attack said that Oliveira shot selectively to immobilize girls while shot boys simply to kill them .
0 0

0.6962025316455697
The second board revision has true JAMMA and also has a switch to select between JAMMA output as well as MVS output , which is stereo sound .
The second board revision is real JAMMA and also has a switch to select between JAMMA output as well as MVS output , which has stereo sound .
0 0

0.7
Born in Dublin about 1582 , he was second but third surviving son of Arland Ussher and his wife Margaret .
Born in Dublin in 1582 , he was the third but second surviving son of Arland Ussher and his wife Margaret .
1 0

0.691358024691358
Another memorable ruler was Max Franz ( founded in 1784 -- 1794 ) , who ruled the university and the spa quarter of Bad Godesberg .
Another memorable ruler was Max Franz ( reg . 1784 - 1794 ) , who founded the university and the spa quarter of Bad Godesberg .
0 0

0.6951219512195121
For example , to enter a vertical line of 4 cm in length , it is sufficient to draw :
For example , in order to draw a vertical line of 4 cm length , it is sufficient to type :
0 0

0.6987951807228916
In 2014 , when Santa Anita Park was closed the race moved to Hollywood Park Racetrack .
In 2014 , when Hollywood Park Racetrack was closed the race in Santa Anita Park was moved .
0 0

0.7023809523809523
( Don Wyatt had been in the penguins in 1956 , and both Eddie and Ray had been with Ray in the later Colts / Fortunes . )
( Don Wyatt had been in the Penguins in 1956 and both Eddie and Ray had been in the later Colts/Fortunes with Ray . )
1 1

0.7058823529411765
This circuit is said to have appeared in the earliest evolution of the reptilian brain and corresponds to the triune brain of invertebrate brain theory .
This circuit is supposed to have appeared in the earliest evolution of the invertebrate brain and corresponds to the reptile brain of the triune - brain theory .
0 0

0.7093023255813954
Lemmings , by contrast , are conspicuously colored and behave aggressively towards predators and even human observers .
By contrast , the lemmings are strikingly colored and behave aggressively towards predators and even human observers .
1 1

0.7126436781609196
It is located in Shawnee Township and adjacent to Duchouquet Township in Allen County .
It is located in Shawnee Township and is adjacent to Duchouquet Township in Allen County .
1 1

0.7159090909090909
He was asked for his opinion about the books '' Mission to Moscow `` by Joseph E. Davies and '' One World `` by Wendell Willkie .
He was asked about his opinions on the books '' Mission to Moscow `` by Wendell Willkie and '' One World `` by Joseph E. Davies .
0 0

0.7191011235955056
The eastern part of Fenghuang Mountain is home to a mountain that has been called Chaoyang since ancient times .
The eastern part of Chaoyang is home to a mountain that has been called the Fenghuang Mountain since the ancient times .
0 0

0.7222222222222222
Guillermo Pérez Roldán defeated Andrés Gómez with 6 -- 0 , 7 -- 6 , 3 -- 6 , 0 - 6 , 6 - 2 .
Andrés Gómez defeated Guillermo Pérez Roldán 6 -- 0 , 7 -- 6 , 3 - 6 , 0 - 6 , 6 - 2
0 0

0.7252747252747253
The 2007 -- 08 Kansas State Wildcats Men 's Basketball Team represents Kansas State University at the 2007 -- 08 College - Basketball - Season .
The 2007 -- 08 Kansas State Wildcats men 's basketball team represented Kansas State University in the 2007 -- 08 college basketball season .
1 1

0.7282608695652174
This was the last AFL Championship to end the season ; the first Super Bowl followed the 1966 season .
This was the last AFL Championship to complete the season , the first Super Bowl followed in the 1966 season .
1 1

0.7311827956989247
Higher is the fifth album , and the fourth studio album , by Ezio , released in 2000 .
Higher is the fourth album and the fifth studio album of Ezio , released in 2000 .
0 0

0.7340425531914894
Finsch 's monitor was only known by Blanche Bay , Ralum and Massawa in New Britain .
The Finsch monitor was known only from Blanche Bay , Ralum and New Britain in Massawa .
0 0

0.7368421052631579

## Final Accuracy: 73.7%